app/javascript/utils/form/final_form/SearchField.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Field } from 'react-final-form'
+import classNames from 'classnames'
+import { Label, FieldError, HelpText } from './Field'
+
+export const SearchField = ({
+  fieldName,
+  label,
+  blockNote,
+  validation,
+  placeholder,
+  maxLength,
+  id,
+  type,
+  helpText,
+  parse,
+  format,
+  disabled = false,
+  isClearable = false,
+  formatOnBlur = false,
+  isDirty = false,
+  onClearClick
+}) => (
+  <Field
+    name={fieldName}
+    validate={validation}
+    format={format}
+    formatOnBlur={formatOnBlur && isDirty}
+  >
+    {({ input, meta }) => {
+      const hasValue = !!input.value
+      return (
+        <>
+          <div
+            className={classNames(
+              'search',
+              label && 'form-group',
+              (meta.error && meta.touched && 'error') || ''
```

Comment:
I think the preferred way to use classNames with conditional classes is like this:
```
classNames(
  'search',
  {
    'form-group': label,
    error: meta.error && meta.touched
  }
)
```

Reply:
thanks for flagging! I had just copypastad from Field.js. Updated there too